apps/etterlatte-saksbehandling-ui/client/src/utils/formattering.ts

```diff
@@ -19,3 +20,16 @@ export const formaterTidspunkt = (dato: Date) => format(dato, 'HH:mm').toString(
 export const formaterStringTidspunkt = (dato: string) => format(new Date(dato), 'HH:mm').toString()
 
 export const formaterDatoMedTidspunkt = (dato: Date) => format(new Date(dato), 'dd.MM.yyyy HH:mm').toString()
+
+export const formaterBehandlingstype = (behandlingstype?: IBehandlingsType): string => {
+  switch (behandlingstype) {
+    case IBehandlingsType.FØRSTEGANGSBEHANDLING:
+      return 'Førstegangsbehandling'
+    case IBehandlingsType.REVURDERING:
+      return 'Revurdering'
+    case IBehandlingsType.MANUELT_OPPHOER:
+      return 'Manuelt opphør'
+    default:
+      return 'Behandling'
```

Comment:
Litt rart at `Behandlingstype` kan være `null`/`undefined` på en `behandling`? Ser for meg at den typen kanskje ikke bør være optional?

Reply:
nvm, jeg som bare stolte på noen feile typer. fikser den til å være strict